tests/cases/ustruct/LV_NeoHookean_passive_genBC/genBC_svFSIplus/src/GenBC.f

```diff
@@ -0,0 +1,232 @@
+c     Created by Mahdi Esmaily Moghadam 05-25-2011
```

Comment:
General comment for GenBC: I personally do not like to mix Fortran and C++ code, especially in view of the fact that GenBC will be soon deprecated. As soon as svZeroDSolver will be available, we should replace these test cases accordingly. @aabrown100-git could you open an issue for this?

Reply:
Done https://github.com/SimVascular/svFSIplus/issues/203